### PR TITLE
Moved Reorder.svg into a js file

### PR DESCRIPTION
**Pull request by keattang**
Now using the js file. 
Allows for setting the colour via the fill style. 
Also fixed a bug when e.dataTransfer is undefined

### CODE REVIEW

File: lib/ViewModels/NowViewingTabViewModel.js [hunk 1]
```diff
@@ -118,7 +120,11 @@ NowViewingTabViewModel.prototype.dragStart = function(viewModel, e) {
         e.preventDefault();
     }, false);
 
-    e.dataTransfer.setData('text', 'Dragging a Now Viewing item.');
+    if (defined(e.dataTransfer)) {
+        e.dataTransfer.setData('text', 'Dragging a Now Viewing item.');
+    } else {
+        e.originalEvent.dataTransfer.setData('text', 'Dragging a Now Viewing item.');
```
Comment (kring) [inline lib/ViewModels/NowViewingTabViewModel.js:126]: Is this because you guys are using jQuery?

Comment (keattang) [inline lib/ViewModels/NowViewingTabViewModel.js:126]: I wasn't sure why it was occurring but we are using jQuery

Comment (kring) [inline lib/ViewModels/NowViewingTabViewModel.js:126]: Yeah jQuery inserts itself into all drag events for some reason.  It's good stuff. :)

**Comment by kring** [comment]
I tweaked this to maintain the previous color of the drag handle.  You can override it by setting `@now-viewing-drag-handle-color`.

**Pull request merged**